test/ask_web/controllers/folder_controller_test.exs

```diff
@@ -163,7 +163,7 @@ defmodule AskWeb.FolderControllerTest do
                    "name" => wave.name,
                    "description" => nil,
                    "project_id" => project.id,
-                   "state" => "not_ready",
+                   "state" => to_string(:not_ready),
```

Comment:
I'd keep the string. Be them params sent to the controller or transformed to JSON, it's a mere `"not_ready"` string. It should just work. Same below, as well as in ShortLinkControllerTest and SurveyControllerTest.